corehq/apps/sms/api.py

```diff
@@ -693,7 +693,7 @@ def create_billable_for_sms(msg, delay=True):
     if not isinstance(msg, SMS):
         raise Exception("Expected msg to be an SMS")
 
-    if not msg.domain:
+    if not settings.ENTERPRISE_MODE or not msg.domain:
```

Comment:
should that be `if settings.ENTERPRISE_MODE`?

nit: might be better to add a method
```
def create_billable_for_sms_if_applicable(msg, delay=True):
    if settings.ENTERPRISE_MODE:
        return
    _create_billable_for_sms(msg)
```
and call that instead.

Reply:
```suggestion
    if settings.ENTERPRISE_MODE or not msg.domain:
```